<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw</class>
 <widget class="QMainWindow" name="mw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>566</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>751</width>
    <height>566</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2UPW | url shortener</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/images/assets/favicon.ico</normaloff>:/images/assets/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-image: url(:/images/assets/wave-app.png);
	background-color:#e9ebef;
}
QGroupBox{
	background:#fff;
	border-radius:6px;
}
QPushButton{
	color:#fff;
	background-color:#0067f4;
}
QPushButton{
	border-radius:19px;
}
QPushButton:hover{
	background-color:rgb(0, 93, 216);
}
QPushButton:pressed{
	border:1px solid #212121;
}
QProgressBar{
	border-radius:3;
	border:1px solid;
	border-color: #fff;
	background: #fff;
}
QProgressBar::chunk:horizontal {
	border-radius:2;
	background:#0067f4;
}
#btn_website{
	color:#212121;
	border-radius:15px;
	background-color:#fff;
}
#btn_website:hover{
	background-color: rgb(241, 241, 241);
}
#btn_developer{
	color:#212121;
	border-radius:15px;
	background-color:#fff;
}
#btn_developer:hover{
	background-color: rgb(241, 241, 241);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_short">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>shorten</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>16</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#fff;
background-image: url(:/images/assets/url-app.png);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="long_url">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>16</y>
       <width>351</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color:#212121;
border-bottom:1px solid rgb(0, 103, 244);
font-size: 14px;</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Paste a long url</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2UPW | url shortener</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="outpanel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_copy">
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-bottom:2px;
padding-right:2px;</string>
     </property>
     <property name="text">
      <string>copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_open_link">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-top:3px;
padding-left:3px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/images/assets/forward.png</normaloff>:/images/assets/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="short_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>351</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#fff;
color:#212121;
border-bottom:1px solid rgb(0, 103, 244);
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>https://2u.pw/sdasdajsd</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>305</y>
      <width>530</width>
      <height>6</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_developer">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Developer</string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/images/assets/013-twitter-1.png</normaloff>:/images/assets/013-twitter-1.png</iconset>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
